### ISSUE TITLE
Changing x-select options gives `Cannot read property 'unregisterOption'` of undefined error

### ISSUE THREAD

**Issue by umairabid** [thefrontside/emberx-select#79]
I am using `x-select` in the following way to render respective states of selected country,  

``` coffeescript

.large-3.columns
  x-select value=model.country class="formcustominput"
    each model.countries
      x-option value=this | #{this}
.large-3.columns.left
  x-select value=model.state  class="formcustominput"
    each model.states
      x-option value=this | #{this}
```

To change states when user selects country I have added observer in the the model and whenever country changes I update my `model.states` but when I do that x-select throws an error

**Comment by umairabid** [comment]
fixed after upgrading to ember 13.4

**Comment by Robdel12** [comment]
Yeah due to ember bugs x-select is only Ember 1.13.4+
